app/Repositories/Eloquent/Repository2.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Repositories\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Container\Container as App;
+use App\Repositories\Contracts\RepositoryInterface2;
+use Exception;
+
+/**
+ * Class Repository
+ *
+ * @package Bosnadev\Repositories\Eloquent
+ */
+abstract class Repository2 implements RepositoryInterface2
```

Comment:
[rank A] please don't make something like that. We cannot define any classes or files with name as xxxx2.